### PR TITLE
fix Scheduler layout

### PR DESCRIPTION
**Pull request by Kongwei93**
- default view to weekly
- rename today button to focus
- remove 4 days option
- remove margin to fit schedule to page

### CODE REVIEW

File: src/views/Schedule.vue [hunk 1]
```diff
@@ -93,7 +90,7 @@
 export default {
   data: () => ({
     focus: "",
-    type: "month",
+    type: "week",
```
Comment (Jaimeloeuf) [inline src/views/Schedule.vue:93]: change to a better name bah, like defaultViewType or smth

File: src/views/Schedule.vue [hunk 2]
```diff
@@ -157,7 +154,7 @@ export default {
     },
     monthFormatter() {
       return this.$refs.calendar.getFormatter({
-        timeZone: "UTC",
+        timeZone: "GMT",
```
Comment (Jaimeloeuf) [inline src/views/Schedule.vue:157]: Why GMT here?

File: src/views/Schedule.vue [hunk 3]
```diff
@@ -245,8 +242,8 @@ export default {
 </script>
 
 <style scoped>
-#schedule {
+/* #schedule {
   margin: 4em;
   margin-top: 2em;
-}
+} */
```
Comment (Jaimeloeuf) [inline src/views/Schedule.vue:248]: Delete if not needed bah

File: src/views/Schedule.vue (hunk 2 above)
Comment (Kongwei93) [reply src/views/Schedule.vue:157]: cause SG timezone ma or are we suppose to leave it at UTC?

File: src/views/Schedule.vue (hunk 1 above)
Comment (Kongwei93) [reply src/views/Schedule.vue:93]: the default is for alot of other variables also. only the focus line is to set the default view type

**Pull request merged**